.github/workflows/build-without-test.yml

```
@@ -0,0 +1,29 @@
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up JDK 17
+      uses: actions/setup-java@v4
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)